news/2023/2023-07-06-introducing-new-beatmap-pack-categories.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Introducing New Beatmap Pack Categories"
+date: 2023-07-06 09:00:00 +0000
+---
+
+Check out the latest and greatest new beatmap pack categories!
+
+![](/wiki/shared/news/2023-07-06-introducing-new-beatmap-pack-categories/banner.png)
+
+I like maps. You like maps. We all like maps.
+
+For this reason, the mad scientists over at osu! labs have been synthesizing new ways to pump high-quality beatmaps directly into your bloodstream, and their latest creations are sure to excite you!
+
+Introducing three new categories of mappack!
```

Comment:
Should be plural and two words I think
```suggestion
Introducing three new categories of map packs!
```

Reply:
'three new categories of map packs' feels like the plurals in there are wrong, could probably avoid that entirely by saying 'three new categories of beatmap pack' instead.